### ISSUE TITLE
Try to finetune pretrained networks

### ISSUE THREAD

**Issue by chriskamphuis** [chriskamphuis/ISMI-Project#24]
Try to vary the batchsize and learning rate

**Comment by Chriskamphuis** [comment]
Also try SGD with some momentum
